.circleci/config.yml

```diff
@@ -1,9 +1,7 @@
 anchors:
-  - &latest-xcode "11.7.0"
-  - &latest-ios   "13.7"
-  - &min-ios      "12.2"
-  - &beta-xcode   "12.0.0"
-  - &beta-ios     "14.0"
+  - &latest-xcode "12.1.0"
+  - &latest-ios   "14.1"
+  - &min-ios      "13.5"
```

Comment:
`13.0`?

Reply:
Blame Apple.  The Xcode 12.1 package only comes bundled with the 14.1 and 13.5 runtimes.

Reply:
Gotcha.